addendum.dict

```diff
@@ -393,6 +412,7 @@ wand'ring	𐑢𐑪𐑯𐑛𐑮𐑦𐑙
 Watkins	𐑢𐑪𐑑𐑒𐑦𐑯𐑟
 Webb	𐑢𐑧𐑚
 Westmorland	𐑢𐑧𐑕𐑑𐑥𐑼𐑤𐑩𐑯𐑛
+Whatsapp 𐑢𐑩𐑑𐑕𐑨𐑐
```

Comment:
I'd prefer 𐑢𐑪𐑑𐑕𐑨𐑐 to conform to principle 14 at https://www.shavian.info/spelling/.